javascript/javascript1/week2/main.js

```diff
@@ -0,0 +1,184 @@
+//Part one homework
+console.log("My username on freeCodeCamp is: Nelia11.")
+
+//Flight booking fullname function. Formal fullname 
+let fullname1 = "Nelia";
+let fullname2 = "Levit";
+
+function getFullname(firstname, surname, useFormalName = false, isWoman = false) {
```

Comment:
It is great that you have found a way to make this program work, but I would encourage you to look at it closer and see what logical improvements you can do